tests/test_legacy_detection.py

```diff
@@ -4,6 +4,7 @@
 from nose.tools import assert_raises
 from numpy.testing import (assert_array_almost_equal, assert_array_equal,
                            assert_equal)
+from morph_tool.spatial import point_to_section_segment
 
 import neuror.cut_plane.legacy_detection as tested
```

Comment:
Why not `import neuror.cut_plane.legacy_detection as test_module`? On other tests you use `test_module`. What's the logic?